spec/ibandit/iban_spec.rb

```diff
@@ -185,6 +185,23 @@
         its(:to_s) { is_expected.to eq("") }
       end
 
+      context "and a 5 digit account number" do
```

Comment:
Should we have a test to check it fails for < 5?

Reply:
Added a new test for < 5. Changing the error message unfortunately is not straightforward, I'll add a separate card for that.